Description: Remove calamares icon desktop entry gnome startup hint 
 The GNOME startup hint in the XDG startup desktop file add-calamares-icon
 contains a line "X-GNOME-Autostart-Phase=Initialization" which causes this
 script not to run on KDE on Debian 12.x.
 .
 For more information, please refer to the Debian bug below.
Author: Jonathan Carter <jcc@debian.org>
Bug-Debian: https://bugs.debian.org/1057853


--- lingmo-installer-config-12.0.9.orig/conf/calamares-desktop-icon.desktop
+++ lingmo-installer-config-12.0.9/conf/calamares-desktop-icon.desktop
@@ -5,5 +5,4 @@ Exec=add-calamares-desktop-icon
 StartupNotify=false
 NoDisplay=true
 
-X-GNOME-Autostart-Phase=Initialization
 X-KDE-autostart-phase=1
